packages/core/useElementBySelector/index.ts

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line no-restricted-imports
+import { ref as deepRef, onMounted } from 'vue'
+
+export function useElementBySelector(selector: string) {
```

Comment:
If we want to merge this we should have reactivity

```suggestion
export function useElementBySelector(selector: MaybeRefOrGetter<string>) {
```

Reply:
Have now added reactive support for the selector argument. Great idea thanks 🙏🏻